### PR TITLE
Jon/ui4/fix/post-upgrade-styling

### PR DESCRIPTION
**Pull request by Jon-edge**
TextInput spacing for the modal not addressed in this PR.

image

image

image

image

image

image

image

Screenshot 2024-01-10 at 3 44 56 PM

Screenshot 2024-01-10 at 3 34 00 PM

Sanity check (SplitRowsView styling update)
Screenshot 2024-01-10 at 3 35 06 PM

Sanity check (RowUi4):
image
image
image
image
Screenshot 2024-01-10 at 4 52 32 PM
image

### CHANGELOG

Does this branch warrant an entry to the CHANGELOG?

- [ ] Yes
- [x] No

### Dependencies

 none

### Requirements

If you have made **any** visual changes to the GUI. Make sure you have:

- [x] Tested on iOS device
- [x] Tested on Android device
- [ ] Tested on small-screen device (iPod Touch)
- [ ] Tested on large-screen device (tablet)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206315466361667

### CODE REVIEW

**Comment by paullinator** [comment]
Since this PR touches RowUi4, did you visually check all places where RowUi4 is used? The screenshots only show the fixed scenes?

File: src/components/modals/CountryListModal.tsx [hunk 1]
```diff
@@ -32,14 +33,11 @@ export const CountryListModal = ({ countryCode = getCountry() ?? 'US', bridge }:
...
     return (
-      <SelectableRow
-        arrowTappable
-        icon={<FastImage source={source} style={styles.image} />}
-        subTitle={alpha}
-        title={name}
-        onPress={() => bridge.resolve(alpha)}
-      />
+      <CardUi4 marginRem={[0.5, 0, 0.5, 0]}>
+        <RowUi4 title={name} body={alpha} icon={<FastImage source={source} style={styles.image} />} onPress={() => bridge.resolve(alpha)} />
```
Comment (paullinator) [inline src/components/modals/CountryListModal.tsx:39]: Using RowUi4 changes the design altogether. These rows do not mimic the design of the original Tile which RowUi4 uses. 

The design should have the top Text is `primaryText` color and rem=1 while the second row is smaller with the secondary text color.

**Review by paullinator — CHANGES_REQUESTED** [review]

File: src/components/modals/CountryListModal.tsx (hunk 1 above)
Comment (Jon-edge) [reply src/components/modals/CountryListModal.tsx]: Reverted, retaining original SelectableRow styling.

File: src/components/themed/SelectableRow.tsx [hunk 2]
```diff
@@ -12,35 +11,39 @@ interface Props {
...
-            ) : null}
-          </View>
-          <IonIcon size={theme.rem(1.5)} color={arrowTappable ? theme.iconTappable : theme.icon} name="chevron-forward-outline" style={styles.chevron} />
+  return (
+    <CardUi4 onPress={onPress} marginRem={marginRem}>
+      <View style={styles.rowContainer}>
+        {/* HACK: Keeping the iconContainer instead of CardUi4's built-in icon prop because the prop's behavior is inconsistent in legacy use cases */}
+        <View style={styles.iconContainer}>{icon}</View>
+        <View style={styles.textContainer}>
+          <EdgeText numberOfLines={1} disableFontScaling>
```
Comment (paullinator) [inline src/components/themed/SelectableRow.tsx:35]: why did you add `disableFontScaling`. This would help with long country names

File: src/locales/en_US.ts [hunk 3]
```diff
@@ -619,6 +619,8 @@ const strings = {
   title_add_token: 'Add Token',
   title_password_recovery: 'Password Recovery',
   title_plugin_buy: 'Buy Cryptocurrency',
+  title_select_region: 'Select Region:',
```
Comment (paullinator) [inline src/locales/en_US.ts:622]: No need for ":"

**Review by paullinator — APPROVED** [review]
Approved but with the request you remove `disableFontScaling` unless you have a very good reason not to

**Comment by paullinator** [comment]
Broken visuals attached. 

1. Learn and Promocard do not align on the left side. Need to fix while keeping the dots centered
2. Markets preview has totally broken spacing

Screenshot 2024-01-10 at 22 17 32
Screenshot 2024-01-10 at 22 17 40
